src/Analyser/MutatingScope.php

```diff
@@ -574,6 +574,27 @@ public function getDefinedVariables(): array
 		return $variables;
 	}
 
+	/**
+	 * @api
+	 * @return array<int, string>
```

Comment:
why not `list<string>` ?

Reply:
Copied from getDefinedVars

Reply:
Fixed: https://github.com/phpstan/phpstan-src/pull/3530/files

Reply:
Great 🎉